src/main/java/com/redhat/cloud/policies/app/rest/AdminService.java

```diff
@@ -135,6 +139,10 @@ public Response setAdminDown(@QueryParam("status") Optional<String> status) {
     @Transactional
     public Response syncToEngine(@QueryParam("token") String token) {
 
+        if (lightweightEngineConfig.isEnabled()) {
+            return Response.status(503).entity("Sync unavailable when the lightweight engine is enabled").build();
```

Comment:
a very simple unit test would be great so we have something like an insurance when we change something somewhere in the code.

Reply:
This is a perfect occasion to show one of the benefits of `LightweightEngineConfig` 😃

Reply:
Test added.